src/conditionals/programs_with_decision_points/medium/q7/MainTest.java

```diff
@@ -0,0 +1,94 @@
+package conditionals.programs_with_decision_points.medium.q7;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+  // Parsons
+  public Clause[] testSentence() {
+    TestOption.isInputTest = true;
+    TestOption.defaultInput = "12345";
+    return new Clause[] {
+        new StringLiteral("Enter a Javatopia ZIP code:"),
+        new NewLine(),
+        new PlaceHolder(),
+        new PlaceHolder(),
```

Comment:
Can you try removing one of these PlaceHolder() variables and removing the NewLine() from your test method? They should not be required, so I want to double-check they are working